### PR TITLE
Release notes and Changelog for 2.0

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: RELEASE.rst [hunk 1]
```diff
@@ -1,44 +1,60 @@
...
+* Various fixes and clarifications to pixel indexing in the map subpackage.
+* Standidisation of specifying rectangles in coordinate space in the ``submap`` and ``draw_rectangle`` methods of ``GenericMap``.
+* A HTML quicklook preview of ``GenericMap`` and ``MapSequence`` which can be accessed with the new ``quicklook()`` method.
+  This is also the default display in Jupyter notebooks.
+* Integration of differential rotation into the sunpy coordinate framework.
+  This enables, amongst other things, the warping of images with the ``reproject`` package and the plotting of rotated grid lines with ``WCSAxes``.
 
-See `What's New in SunPy 1.1 <https://docs.sunpy.org/en/stable/whatsnew/1.1.html>`__ for more details and the `Full Changelog <https://docs.sunpy.org/en/stable/whatsnew/changelog.html>`__ for the full list of over 100 changes in 1.1.
+See `What's New in SunPy 2.0 <https://docs.sunpy.org/en/stable/whatsnew/2.0.html>`__ for more details and the `Full Changelog <https://docs.sunpy.org/en/stable/whatsnew/changelog.html>`__ for the full list of over 100 changes in 1.1.
```
Comment (samaloney) [inline RELEASE.rst:19]:
```suggestion
See `What's New in SunPy 2.0 <https://docs.sunpy.org/en/stable/whatsnew/2.0.html>`__ for more details and the `Full Changelog <https://docs.sunpy.org/en/stable/whatsnew/changelog.html>`__ for the full list of over 100 changes in 2.0
```

File: RELEASE.rst [hunk 2]
```diff
@@ -1,44 +1,60 @@
-The SunPy project is happy to announce the release of SunPy 1.1!
+The SunPy project is happy to announce the release of SunPy 2.0!
 SunPy is an open-source Python library for Solar Physics data analysis and visualization.
 
-This release is our first major release post-1.0 and part of our new `release pattern <https://github.com/sunpy/sunpy-SEP/blob/master/SEP-0009.md>`__.
+This release is our second long(er) term support release, that we will be supporting with bug fixes until 3.0 in roughly a years time.
```
Comment (ayshih) [inline RELEASE.rst:4]:
```suggestion
This release is our second long(er) term support release, that we will be supporting with bug fixes until 3.0 in roughly a year's time.
```

File: RELEASE.rst [hunk 3]
```diff
@@ -1,44 +1,60 @@
...
+With this release the 1.0 and 1.1 releases will no longer recieve bug fixes and we encourage everyone to upgrade to 2.0
```
Comment (ayshih) [inline RELEASE.rst:5]:
```suggestion
With this release, the 1.0 and 1.1 releases will no longer recieve bug fixes and we encourage everyone to upgrade to 2.0.
```

File: RELEASE.rst [hunk 4]
```diff
@@ -1,44 +1,60 @@
...
 The major highlights of this release are:
 
-* The `~sunpy.coordinates` subpackage now supports four additional coordinate frames (HCI, HEE, GSE, and GEI).
-* A new subpackage `sunpy.data.data_manager` has been added to support versioned data for functions and methods.
-* Support in `sunpy.map` and `sunpy.net` for the SUVI instrument on GOES satellites.
-* Initial support for WISPR data from Parker Solar Probe in `sunpy.map`.
-* The import times for `sunpy` and some subpackages are significantly shorter, with no loss of functionality.
+* `~sunpy.net.Fido` now supports tab completion of search attributes.
+  This allows you to do ``a.Instrument.AIA``, and print ``a.Instrument`` to see the list of known supported instruments.
+* `~sunpy.instr.aia.aiaprep` has been deprecated in favour of the functionality in the `aiapy <https://aiapy.readthedocs.io/>`__ package.
```
Comment (ayshih) [inline RELEASE.rst:11]:
```suggestion
* `~sunpy.instr.aia.aiaprep` has been deprecated in favor of the functionality in the `aiapy <https://aiapy.readthedocs.io/>`__ package.
```

File: RELEASE.rst [hunk 5]
```diff
@@ -1,44 +1,60 @@
...
+* Various fixes and clarifications to pixel indexing in the map subpackage.
+* Standidisation of specifying rectangles in coordinate space in the ``submap`` and ``draw_rectangle`` methods of ``GenericMap``.
```
Comment (ayshih) [inline RELEASE.rst:13]:
```suggestion
* Standardization of specifying rectangles in coordinate space in the :meth:`~sunpy.map.GenericMap.submap` and :meth:`~sunpy.map.GenericMap.draw_rectangle` methods of `~sunpy.map.GenericMap`.
```

File: RELEASE.rst [hunk 6]
```diff
@@ -1,44 +1,60 @@
...
+* A HTML quicklook preview of ``GenericMap`` and ``MapSequence`` which can be accessed with the new ``quicklook()`` method.
```
Comment (ayshih) [inline RELEASE.rst:14]:
```suggestion
* HTML quicklook previews of `~sunpy.map.GenericMap` and `~sunpy.map.MapSequence` instances are available with the new :meth:`~sunpy.map.GenericMap.quicklook` and :meth:`~sunpy.map.MapSequence.quicklook` methods, respectively.
```

File: RELEASE.rst [hunk 7]
```diff
@@ -1,44 +1,60 @@
...
 
-* The `~sunpy.coordinates` subpackage now supports four additional coordinate frames (HCI, HEE, GSE, and GEI).
-* A new subpackage `sunpy.data.data_manager` has been added to support versioned data for functions and methods.
-* Support in `sunpy.map` and `sunpy.net` for the SUVI instrument on GOES satellites.
-* Initial support for WISPR data from Parker Solar Probe in `sunpy.map`.
-* The import times for `sunpy` and some subpackages are significantly shorter, with no loss of functionality.
+* `~sunpy.net.Fido` now supports tab completion of search attributes.
+  This allows you to do ``a.Instrument.AIA``, and print ``a.Instrument`` to see the list of known supported instruments.
+* `~sunpy.instr.aia.aiaprep` has been deprecated in favour of the functionality in the `aiapy <https://aiapy.readthedocs.io/>`__ package.
+* Various fixes and clarifications to pixel indexing in the map subpackage.
```
Comment (ayshih) [inline RELEASE.rst:12]:
```suggestion
* Various fixes and clarifications to pixel indexing in the `sunpy.map` subpackage.
```

File: RELEASE.rst [hunk 8]
```diff
@@ -1,44 +1,60 @@
...
+* Standidisation of specifying rectangles in coordinate space in the ``submap`` and ``draw_rectangle`` methods of ``GenericMap``.
+* A HTML quicklook preview of ``GenericMap`` and ``MapSequence`` which can be accessed with the new ``quicklook()`` method.
+  This is also the default display in Jupyter notebooks.
+* Integration of differential rotation into the sunpy coordinate framework.
+  This enables, amongst other things, the warping of images with the ``reproject`` package and the plotting of rotated grid lines with ``WCSAxes``.
```
Comment (ayshih) [inline RELEASE.rst:17]:
```suggestion
  This enables, amongst other things, the warping of images with the `reproject <https://reproject.readthedocs.io/>`__ package and the plotting of rotated grid lines with :ref:`WCSAxes <astropy:wcsaxes>`.
```

File: RELEASE.rst [hunk 9]
```diff
@@ -1,44 +1,60 @@
...
-* Initial support for WISPR data from Parker Solar Probe in `sunpy.map`.
-* The import times for `sunpy` and some subpackages are significantly shorter, with no loss of functionality.
+* `~sunpy.net.Fido` now supports tab completion of search attributes.
+  This allows you to do ``a.Instrument.AIA``, and print ``a.Instrument`` to see the list of known supported instruments.
+* `~sunpy.instr.aia.aiaprep` has been deprecated in favour of the functionality in the `aiapy <https://aiapy.readthedocs.io/>`__ package.
+* Various fixes and clarifications to pixel indexing in the map subpackage.
+* Standidisation of specifying rectangles in coordinate space in the ``submap`` and ``draw_rectangle`` methods of ``GenericMap``.
+* A HTML quicklook preview of ``GenericMap`` and ``MapSequence`` which can be accessed with the new ``quicklook()`` method.
+  This is also the default display in Jupyter notebooks.
+* Integration of differential rotation into the sunpy coordinate framework.
```
Comment (ayshih) [inline RELEASE.rst:16]:
```suggestion
* Integration of differential rotation into the `sunpy.coordinates` framework.
```

File: RELEASE.rst [hunk 10]
```diff
@@ -1,44 +1,60 @@
...
-* Support in `sunpy.map` and `sunpy.net` for the SUVI instrument on GOES satellites.
-* Initial support for WISPR data from Parker Solar Probe in `sunpy.map`.
-* The import times for `sunpy` and some subpackages are significantly shorter, with no loss of functionality.
+* `~sunpy.net.Fido` now supports tab completion of search attributes.
+  This allows you to do ``a.Instrument.AIA``, and print ``a.Instrument`` to see the list of known supported instruments.
+* `~sunpy.instr.aia.aiaprep` has been deprecated in favour of the functionality in the `aiapy <https://aiapy.readthedocs.io/>`__ package.
+* Various fixes and clarifications to pixel indexing in the map subpackage.
+* Standidisation of specifying rectangles in coordinate space in the ``submap`` and ``draw_rectangle`` methods of ``GenericMap``.
+* A HTML quicklook preview of ``GenericMap`` and ``MapSequence`` which can be accessed with the new ``quicklook()`` method.
+  This is also the default display in Jupyter notebooks.
```
Comment (ayshih) [inline RELEASE.rst:15]:
```suggestion
  This is also the default display in `Jupyter <https://jupyter.org/>`__ notebooks.
```

File: RELEASE.rst [hunk 11]
```diff
@@ -1,44 +1,60 @@
...
+* Integration of differential rotation into the sunpy coordinate framework.
+  This enables, amongst other things, the warping of images with the ``reproject`` package and the plotting of rotated grid lines with ``WCSAxes``.
 
-See `What's New in SunPy 1.1 <https://docs.sunpy.org/en/stable/whatsnew/1.1.html>`__ for more details and the `Full Changelog <https://docs.sunpy.org/en/stable/whatsnew/changelog.html>`__ for the full list of over 100 changes in 1.1.
+See `What's New in SunPy 2.0 <https://docs.sunpy.org/en/stable/whatsnew/2.0.html>`__ for more details and the `Full Changelog <https://docs.sunpy.org/en/stable/whatsnew/changelog.html>`__ for the full list of over 100 changes in 2.0.
 
-This release of SunPy contains 1137 commits in 242 merged pull requests closing 106 issues from 24 people, 10 of which are first time contributors to SunPy.
+
+This release of SunPy contains 1044 commits in 290 merged pull requests closing 144 issues from 33 people, 16 of which are first time contributors to SunPy.
```
Comment (ayshih) [inline RELEASE.rst:22]:
```suggestion
This release of SunPy contains 1044 commits in 290 merged pull requests closing 144 issues from 33 people, 16 of which are first-time contributors to SunPy.
```

**Comment by Cadair** [comment]
Backported manually, and pushed directly to 2.0 to save some CI time.

**Pull request merged**